test/slow/timeouts.js

```diff
@@ -0,0 +1,44 @@
+const test = require('brittle')
+const UDX = require('../')
+// const { makeTwoStreams } = require('./helpers')
+
+test('default firewall - same socket', async function (t) {
+  t.plan(1)
+
+  const u = new UDX()
+
+  const socket = u.createSocket()
+  socket.bind(0)
+
+  const a = u.createStream(1)
+  const b = u.createStream(2)
+
+  b.on('data', function (data) {
+    t.fail('default firewall should not allow to receive data')
+  })
+
+  a.on('error', function (error) {
+    t.is(error.code, 'ETIMEDOUT')
+
+    socket.close()
+  })
+
+  a.connect(socket, 2, socket.address().port)
+  a.write(Buffer.from('hello'))
+})
+
+/* test('default firewall - different sockets', async function (t) {
+  t.plan(1)
+
+  const [a, b] = makeTwoStreams(t)
+
+  b.on('data', function (data) {
+    t.fail('default firewall should not allow to receive data')
+  })
+
+  a.on('error', function (error) {
+    t.is(error.code, 'ETIMEDOUT')
+  })
+
+  a.write(Buffer.from('hello'))
+}) */
```

Comment:
@LuKks how come this is commented out?

Reply:
It's not necessarily intended to be merged, it's in case you know why it doesn't work?

Reply:
let me see